internal/model/manifest.go

```diff
@@ -104,24 +97,33 @@ func (m Manifest) Dockerignores() []Dockerignore {
 func (m Manifest) LocalPaths() []string {
 	if sbInfo := m.StaticBuildInfo(); !sbInfo.Empty() {
```

Comment:
Could we transform this in to a case statement?

Reply:
not really, cuz we're looking at the type of two different fields :-/